0_intro_to_jax_for_ml.ipynb

```diff
@@ -0,0 +1,2081 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #6.    # Run it on a single vecor
</pre><p>I think this single vector example might actually cause more confusion than help. Since I don't think this issue will actually come up. Not sure though...</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deep-learning-indaba/indaba-pracs-2022/pull/4/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='95'/>

Reply:
<p>Valid. Let's see what other tutors say.</p>